addenda05_test.go

```diff
@@ -108,48 +108,37 @@ func BenchmarkAddenda05String(b *testing.B) {
 func TestValidateAddenda05RecordType(t *testing.T) {
 	addenda05 := mockAddenda05()
 	addenda05.recordType = "63"
-	if err := addenda05.Validate(); err != nil {
-		if e, ok := err.(*FieldError); ok {
-			if e.FieldName != "recordType" {
-				t.Errorf("%T: %s", err, err)
-			}
-		}
+	err := addenda05.Validate()
+	// TODO: are we not expecting any errors here?
```

Comment:
same here, we just need to return the fieldError

	if addenda05.recordType != "7" {
		fieldError("recordType", NewErrRecordType(7), addenda05.recordType)
	}